crates/sui-tool/src/commands.rs

```diff
@@ -709,6 +710,9 @@ impl ToolCommand {
                     }
                 };
 
+                if let Err(e) = check_completed_snapshot(&snapshot_store_config, epoch).await {
+                    panic!("Aborting snapshot restore: {}", e);
```

Comment:
Could we add here something like "this may be because this snapshot has not yet begun or finished uploading"?